easybuild/easyconfigs/n/nghttp3/nghttp3-0.6.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'nghttp3'
+version = '0.6.0'
+
+homepage = 'https://github.com/ngtcp2/nghttp3'
+description = """ nghttp3 is an implementation of RFC 9114 HTTP/3 
+mapping over QUIC and RFC 9204 QPACK in C.
+It does not depend on any particular QUIC transport implementation."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'ngtcp2'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+
+checksums = [
+    '3cd5fa107541a2dd11b87a84f251a98fae4c7b67df9d78be7343e4adce540b7f',  # v0.6.0.tar.gz
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('pkg-config', '0.29.2'),
+    ('CUnit', '2.1-3'),
```

Comment:
Should be `2.1.3` (resolved via https://github.com/easybuilders/easybuild-easyconfigs/pull/16972)?

Reply:
nope @smoors did a mistake renaming the title #16972 https://sourceforge.net/projects/cunit/files/CUnit/2.1-3/

Reply:
fixed